wiki/en/misc/1-index.html

```diff
@@ -17,7 +17,7 @@
   <div class="fx-col-100-xs">
     <div itemprop="abstract">
       <h2>What is Jamulus?</h2>
-       Jamulus is for playing, rehearsing, or just jamming with your friends, your band or just anyone you find online. With high quality, low-latency sound on a normal broadband connection, it's easy to play together remotely and in time. <a href="wiki/Getting-Started" target="_blank" rel="noreferrer">Download it here</a>!
+       Jamulus is a software for playing, rehearsing, or jamming with your friends, your band or anyone you find online. Play together remotely in time with high quality, low-latency sound on a normal broadband connection. <a href="wiki/Getting-Started" target="_blank" rel="noreferrer">Download it here</a>!
```

Comment:
```suggestion
       Jamulus is software that lets you play, rehearse, or jam with your friends, your band or anyone you find online. Play together remotely in time with high quality, low-latency sound on a normal broadband connection. <a href="wiki/Getting-Started" target="_blank" rel="noreferrer">Download it here</a>!
```

Reply:
I still think mentioning "software" is unnecessary.  "With Jamulus, you can play, rehearse, or jam with your friends..."

Reply:
True. If the site still makes it clear that Jamulus is a software (and Google gets that) it should be fine.

Reply:
>Jamulus is a software

This remains incorrect English.

Reply:
Ok. Sorry. I meant to say that it should be clear that Jamulus is not a device, service,... 
do we have the "a software" mistake anywhere else?

Reply:
All looks okay:
```
$ grep -r 'a software'
_posts/2020-10-19-Software-Synth.md:This article will show you how to use a software synth with Jamulus.
_posts/2020-10-19-Software-Synth.md:We will look at the concept of playing a software synth on Linux but other operating systems should work similarly.
README.md:1. Once we’ve agreed the changes can go live (usually just before a software release), we then create GitHub issues for each language, tagged for that release. You can ask questions about the work there.
_translator-files/po/nl/Installation-for-iOS.po:msgstr "**Opmerking**: we bieden een niet-ondertekend .ipa-bestand dat je moet ondertekenen en vervolgens op je apparaat moet installeren. Als je niet gejailbreakt bent (en AppSync hebt geïnstalleerd), heb je mogelijk extra software nodig om Jamulus te installeren. Er zijn meerdere manieren om dat te doen, maar deze gids legt alleen de meest standaardaanpak uit. Houd er rekening mee dat we ervan uitgaan dat je al een (gratis) Apple-ontwikkelaarsaccount hebt. Als je er geen hebt, maak dan een Apple ID aan en [meld je aan voor een ontwikkelaarsaccount](https://developer.apple.com/membercenter). We raden aan om er een aan te maken zonder twee-factor-authenticatie en niet gerelateerd aan je potentiële persoonlijke account. Je moet het wachtwoord invoeren in niet-Apple-software.\n"
LICENSE:    a) The modified work must itself be a software library.
```
> a software synth
> a software release
> heb je mogelijk extra software nodig om Jamulus te installeren
> a software library

All using `software` as a descriptive term for another noun.  Like "a software program".  "Jamulus is a software program" is correct but "Jamulus is a software" is not.  However, it can be used as an abstract noun: "Jamulus is software that allows you to jam with others".

(I checked jamulussoftware/jamulus, too, and no occurrences there.)

Reply:
Ok. Then I think all is good here.